packages/phishing-controller/src/PhishingDetector.test.ts

```diff
@@ -1105,6 +1105,229 @@ describe('PhishingDetector', () => {
       });
     });
   });
+
+  describe('PhishingDetector - isBlocked', () => {
```

Comment:
Can we rename this describe to the name of the public method being tested? In this case `isMaliciousRequestDomain`. Also there's no need to reference the `PhishingDetector` as that is included in the parent describe